api/v1/db/oracledb/parks-dao.js

```diff
@@ -277,10 +277,34 @@ const patchParkById = async (id, body) => {
   }
 };
 
+/**
+ * @summary Get park by owner Id
+ * @param {string} id Unique owner ID
+ * @returns {Promise<object>} Promise object represents a specific owner or return undefined if term
+ *                            is not found
+ */
+const getParkByOwnerId = async (id) => {
```

Comment:
Since an owner can have multiple parks, should this be called `getParksByOwnerId`?